master/buildbot/worker/docker.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, name, password, docker_host, image=None, command=None,
             except ValueError:
                 config.error("Invalid volume definition for docker "
                              "%s. Skipping..." % volume_string)
+                break
```

Comment:
@rutsky I had continue first, but I think failing at the point of parsing would be better :)  will switch